Configuration/RealmSwift/Tests.xcconfig

```diff
@@ -4,6 +4,7 @@ INFOPLIST_FILE = RealmSwift/Tests/RealmSwiftTests-Info.plist;
 PRODUCT_NAME = $(TARGET_NAME);
 SWIFT_OBJC_BRIDGING_HEADER = RealmSwift/Tests/RealmSwiftTests-BridgingHeader.h
 SWIFT_OPTIMIZATION_LEVEL = -Onone;
+OTHER_SWIFT_FLAGS = -DREALM_XCODE_VERSION_$(XCODE_VERSION_ACTUAL);
```

Comment:
I think `XCODE_VERSION_MINOR` would be a better fit than `XCODE_VERSION_ACTUAL` for this since it excludes the z portion of Xcode's x.y.z version number. That will avoid requiring that we update uses of this define for an Xcode 7.3.1 release.